Poker.BE/Poker.BE.Domain/Core/AbstractUser.cs

```diff
@@ -10,7 +10,7 @@ public abstract class AbstractUser
     {
 
         #region Properties
-        protected string UserName { get; set;}
+        public string UserName { get; set;}
         public string Password { get; set; }
         protected Bank UserBank {get; set;}
```

Comment:
👎  i just saw it now - properties shouldn't be `protected`. only `public` (or `internal`). `protected` and `private` are best used for fields and private function.
i guess this code belongs to @arieltobiana or @tomer314  - please note that!